src/recording/macromanager.cpp

```diff
@@ -17,20 +18,33 @@ MacroManager::MacroManager()
             &MacroManager::slotToggleRecording);
 }
 
+void MacroManager::appendHotcueJump(int channel, double origin, double target) {
```

Comment:
```suggestion
void MacroManager::appendHotcueJump(ChannelHandle channel, double origin, double target) {
```
Maybe we should add a method to PlayerManager to get the Deck for a given ChannelHandle.

Reply:
Hm, right now I'd rather store a plain int to enable quick comparisons in the real-time code. But you do have a point here.

Reply:
ChannelHandle is just a lightweight wrapper around an int so we can avoid QString comparisons in realtime code.

Reply:
But what's the advantage over a plain int? Descriptive typing? Couldn't also be achieved using a typedef?

Reply:
Type safety. You can't accidently assign a regular int a ChannelHandle or vice versa.